src/test/features/claim/helpers/featuresBuilder.ts

```diff
@@ -124,6 +124,25 @@ describe('FeaturesBuilder', () => {
     })
   })
 
+  describe('Legal advisor Pilot Feature', () => {
+    it('should add legal advisor eligible to features if principal amount <= 300 and flag is set', async () => {
+      mockFeatureFlag('cmc_legal_advisor', true)
+      const features = await FeaturesBuilder.features(limitDraft, user)
```

Comment:
It is probably best to create the legal advisor limit as separate test data as part of the change as the one used here will be changed to 1000 when you do the next story.